packages/manager/src/features/Linodes/MigrateLinode/ConfigureForm.tsx

```diff
@@ -98,6 +104,17 @@ export const ConfigureForm = React.memo((props: Props) => {
     [backupEnabled, currentLinodeType]
   );
 
+  const linodeIsInEdgeRegion =
+    flags.gecko && currentActualRegion?.site_type === 'edge';
```

Comment:
`linodeIsInEdgeRegion` doesn't really describe the fact that is checks for the feature flag. Maybe we could consider moving the flag check elsewhere?

```suggestion
  const linodeIsInEdgeRegion =  currentActualRegion?.site_type === 'edge';
```